src/azure-cli/azure/cli/command_modules/acs/azuremonitormetrics/helper.py

```diff
@@ -13,6 +13,19 @@
 )
 
 
+def get_subscription_id_from_resource_id(resource_id):
```

Comment:
maybe you could use parse_resource_id (import from msrestazure.tools) to get the sub id

Reply:
Makes sense, Thanks @FumingZhang I have updated the PR to reflect this.